src/GafferUIBindings/PlugAdderBinding.cpp

```diff
@@ -89,13 +92,71 @@ struct PlugMenuSlotCaller
 
 };
 
+struct PlugAdderWrapper : public PlugAdder, IECorePython::Wrapper< PlugAdder >
+{
+
+	PlugAdderWrapper(PyObject *self, StandardNodeGadget::Edge edge ) : PlugAdder( edge ), IECorePython::Wrapper<PlugAdder>( self, this )
+	{
+	}
+
+	bool acceptsPlug( const Gaffer::Plug *connectionEndPoint ) const
+	{
+		IECorePython::ScopedGILLock gilLock;
+		override o = this->get_override( "acceptsPlug" );
+		if( o )
+		{
+			try
+			{
+				return o( Gaffer::PlugPtr( const_cast<Gaffer::Plug *>( connectionEndPoint ) ) );
+			}
+			catch ( error_already_set )
+			{
+				PyErr_Print();
```

Comment:
Ah, makes sense. I grabbed this bit from a piece of code that seemed to be concerned about not crashing Maya and therefor avoiding exceptions. Will fix - thanks for all the other comments as well. I hope I get around to addressing all of them quickly.